drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/ChromeDriverFactory.java

```diff
@@ -120,6 +120,7 @@ public void setChromeOptions(WebDriverConfiguration configuration, DesiredCapabi
         String browser = configuration.getBrowser().toLowerCase();
         if (browser.equals("chromeheadless")) {
             chromeOptions.addArguments("--headless");
+            chromeOptions.addArguments("--window-size=1200,600");
```

Comment:
I'm not sure if setting specific sizes is the best approach - why this size and not bigger or smaller one...? I'm not also sure if this (or other) sizes wouldn't cause unexpected problems on some platforms/PC configurations. 
I can imagine that we could use the size specified in `dimensions` property, otherwise, I wouldn't set there anything.